sources/packages/backend/apps/workers/src/controllers/supporting-user/supporting-user.controller.ts

```diff
@@ -61,6 +67,24 @@ export class SupportingUserController {
         (supportingUser) => supportingUser.id,
       );
       jobLogger.log("Created supporting users.");
+      const application = await this.applicationService.getApplicationById(
```

Comment:
Ensure that all the DB operations from line 62 (createSupportingUsers) happen in transaction and the methods use transaction entity manager. 

For reference here is the sample code. (Sample code)[https://github.com/bcgov/SIMS/blob/363df0d21ea086e3329e0605c9b1aa2147305376/sources/packages/backend/apps/workers/src/controllers/application/application.controller.ts#L129]